lrpar/tests/src/goal

```diff
@@ -0,0 +1,37 @@
+// build.rs, requires grm/lex in files
```

Comment:
I'm not sure what this file is doing. Any hints?

Reply:
Oh, I dunno how this snug in all of a sudden. I was being careful not to add it. This can obviously  go!